conda/core/package_cache_data.py

```diff
@@ -326,7 +326,7 @@ def _make_single_record(self, package_filename):
                 # JsonDecodeError if info/index.json is partially extracted or corrupted
                 #   python 2.7 raises ValueError instead of JsonDecodeError
                 #   ValueError("No JSON object could be decoded")
-                log.debug("unable to read %s\n  because",
+                log.debug("unable to read %s\n  because %r",
                           join(extracted_package_dir, 'info', 'index.json'), e)
```

Comment:
Huh!

Reply:
Yes, I ran into this error while debugging. The missing placeholder causes the exception to be thrown for other reasons and it's veeery misleading.

Reply:
@jaimergp see https://github.com/conda/conda/pull/11679, need to rebase